packages/dds/tree/src/feature-libraries/sequence-field/compose.ts

```diff
@@ -225,9 +226,13 @@ function composeMarks<TNodeChange>(
 }
 
 function updateModifyLike<TNodeChange>(
+    currRev: RevisionTag | undefined,
     curr: Modify<TNodeChange>,
     base: ModifyInsert<TNodeChange> | Modify<TNodeChange> | ModifyReattach<TNodeChange>,
     composeChild: NodeChangeComposer<TNodeChange>,
 ) {
-    base.changes = composeChild([base.changes, curr.changes]);
+    base.changes = composeChild([
+        tagChange(base.changes, undefined),
```

Comment:
Why is `undefined` okay here? I understand that `base.changes` may come to represent changes from multiple revisions but it won't start that way.
For example if we're composing two changes X and Y where X is modifying a point node's x value and Y is modifying the same point node's y value then it seems to be this code will lead to the first (in time) value change not being labeled with a revision.